web/src/features/prompts/server/utils/validation.ts

```diff
@@ -12,17 +12,26 @@ export enum PromptType {
   Text = "text",
 }
 
+export const PromptLabelSchema = z
+  .string()
+  .min(1)
+  .max(20)
+  .regex(
+    /^[a-z0-9_\-.]+$/,
+    "Label must be lowercase alphanumeric with optional underscores, hyphens, or periods",
+  );
```

Comment:
This here could be a regression:
- someone having a prompt name which does not comply with this
- trying to create a new prompt with the existing name
-> would result in an API error

in prod db, we have lots of different prompt names, which do not comply with this.